web/src/utils/sentry.ts

```diff
@@ -1,52 +1,84 @@
+/* eslint-disable no-console */
 import type Sentry from '@sentry/react'
 
+import { FetchError, NotFoundError } from 'api-client'
+
 import buildConfig from '../constants/buildConfig'
 
-const loadSentry = async (): Promise<typeof Sentry> =>
-  import(
-    /* webpackChunkName: "sentry" */
-    '@sentry/react'
-  )
+const sentryEnabled = (): boolean => buildConfig().featureFlags.sentry
```

Comment:
Why not just `const sentryEndable = buildConfig().featureFlags.sentry`?

Reply:
Was that way before, not sure why it was done that way. However, it makes mocking the build config way easier in tests since you can change values in between test. If this is not a function, it keeps the first value (at least that happened to me on native lately).